pkg/apps/blackduck/components/pvc/v1/pvc.go

```diff
@@ -1,9 +1,31 @@
+/*
+Copyright (C) 2019 Synopsys, Inc.
+
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements. See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership. The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License. You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied. See the License for the
+specific language governing permissions and limitations
+under the License.
+*/
+
 package v1
 
 import (
 	"fmt"
 	"github.com/blackducksoftware/horizon/pkg/components"
 	blackduckapi "github.com/blackducksoftware/synopsys-operator/pkg/api/blackduck/v1"
+	"github.com/blackducksoftware/synopsys-operator/pkg/apps/blackduck"
```

Comment:
why it is added?

Reply:
I'm not sure.  I didn't add it so I suspect GoLand did.

Reply:
It is needed, that's why travis on master is failing. I was gonna patch it in PR #704

Reply:
They are needed to compile, but I didn't add them and I'm not sure GoLand did either.  Maybe an issue with the version I branched from